src/dvc_data/cli.py

```diff
@@ -311,13 +311,11 @@ def du(oid: str = typer.Argument(..., allow_dash=True)):
     odb = get_odb()
     oid = from_shortoid(odb, oid)
     obj = load(odb, odb.get(oid).hash_info)
-    if isinstance(obj, HashFile):
-        tree = Tree()
-        tree.add(ROOT, None, obj.hash_info)
-    elif isinstance(obj, Tree):
-        tree = obj
-    else:
+    if not isinstance(obj, HashFile):
         raise AssertionError(f"unknown object of type {type(obj)}")
+
+    tree = obj if isinstance(obj, Tree) else Tree()
+    tree.add(ROOT, None, obj.hash_info)
```

Comment:
@efiop, what do you think `du` (asking about the internal API, not the CLI) should return for the tree? Currently it returns sum of all files in the tree, but does not return size of the tree-hashfile itself.

Reply:
that sounds very reasonable to me not not accound for tree sizes. 👍

Reply:
Just to clarify, what you are saying is that the current `du` implementation is correct, and we should not add the tree hashfile's size to the result, right?

Reply:
@skshetry Correct.